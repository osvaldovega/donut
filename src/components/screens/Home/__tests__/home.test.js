import React from 'react';
import renderer from 'react-test-renderer';
import HomeScreen from '../index';

describe('<HomeScreen />', () => {

  it('renders correctly', () => {
    const tree = renderer.create(<HomeScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
