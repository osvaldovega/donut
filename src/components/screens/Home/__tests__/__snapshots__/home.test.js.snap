// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeScreen /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F4C3C2",
      "color": "#ffffff",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "flex-start",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "column",
        "height": "60%",
        "justifyContent": "center",
        "width": "100%",
      }
    }
  >
    <Image
      resizeMode="contain"
      source={1}
      style={
        Object {
          "height": "80%",
          "width": "80%",
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "column",
        "height": "40%",
        "justifyContent": "flex-start",
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#2f3542",
          "fontFamily": "montserrat",
          "padding": 30,
          "textAlign": "justify",
          "width": "100%",
        }
      }
    >
      This is a simple app, to check the Coinbase Market Information.
    </Text>
  </View>
</View>
`;
